lib/bolt_server/file_cache.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require 'digest'
+require 'fileutils'
+require 'net/http'
+
+module BoltServer
+  class FileCache
```

Comment:
Do we know the memory impact of this / have a way to limit it? Assuming that'll be configurable and added to the 'expire' logic...?

Reply:
the expiry should help with disk space rather then memory usage. Overall I would not expect much memory difference. Streaming files to disk may help(there is a TODO) there.